packages/webview_flutter/example/lib/main.dart

```diff
@@ -7,18 +7,33 @@ import 'package:webview_flutter/webview_flutter.dart';
 
 void main() => runApp(MaterialApp(home: WebViewExample()));
 
-class WebViewExample extends StatelessWidget {
+class WebViewExample extends StatefulWidget {
+  @override
+  State<StatefulWidget> createState() => WebViewExampleState();
+}
+
+class WebViewExampleState extends State<WebViewExample> {
+  WebViewController controller;
+
   @override
   Widget build(BuildContext context) {
     return Scaffold(
       appBar: AppBar(
         title: const Text('Flutter WebView example'),
         // This drop down menu demonstrates that Flutter widgets can be shown over the web view.
-        actions: <Widget>[const SampleMenu()],
+        actions: <Widget>[
+          (controller != null) ? MoveControl(controller) : Container(),
```

Comment:
I'd expect the back/forward buttons to be there even before the WebView loaded.
What do you think about changing the `controller` field to be a Completer, complete it in `onWebViewCreated`, and pass the completer's future to `MoveControl`, Similarly to how it's done in the WebView class: 
https://github.com/flutter/plugins/blob/8603d2d60eecdac2aa75f07e456d28203fe20292/packages/webview_flutter/lib/webview_flutter.dart#L64

Reply:
I didn't know `Completable`! I try to use `Completable`. -> 0fe8df0